packages/components/chip/src/Chip.stories.tsx

```diff
@@ -32,13 +36,29 @@ export default meta
 
 export const Default: StoryFn = _args => <Chip>default chip</Chip>
 
-export const SingleSelectionFilter: StoryFn = _args => {
-  const [activeFilter, setActive] = useState<undefined | string>('number')
+const singleSet = Array.from(
+  new Map(
+    Object.entries({
+      '🥝': ['fruit'],
+      '🍎': ['fruit'],
+      '🍇': ['fruit'],
+      '🍌': ['fruit'],
+      '🍆': ['tubercle'],
+      '🧅': ['tubercle'],
+      '🥔': ['tubercle'],
+      '🌶': ['tubercle'],
+      '🥕': ['tubercle'],
+    })
+  )
+).sort(() => 0.5 - Math.random())
+
+export const SingleSelectionFilter: StoryFn = () => {
+  const [activeFilter, setActive] = useState<undefined | string>('fruit')
 
   return (
     <div className="flex flex-col gap-md">
       <div className="flex flex-row items-start gap-md">
-        {['vowel', 'consonant', 'number', 'symbol'].map(filter => {
+        {['fruit', 'tubercle'].map(filter => {
```

Comment:
it seems weird to me to handle filtering within the `map` method. Imho, it would be more appropriate to first filter the list and then map over it

Reply:
well probably the error is about the naming. Maybe better replace filter variable name to element, cuz the filter is the usage of the elements as children of the Chip.